packages/react-web/src/inbox/useXmtpClient.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  PendingSigningRequestError,
+  UserRejectedError,
+  WalletConnectionError,
+  useActiveWalletController,
+  useInboxKeyStorage,
+} from '@lens-protocol/react';
+import { assertError } from '@lens-protocol/shared-kernel';
+import { IStorage } from '@lens-protocol/storage';
+import { Client, ClientOptions, useClient } from '@xmtp/react-sdk';
+import { useCallback, useState } from 'react';
+
+import { SignerAdapter } from './adapters/SignerAdapter';
+
+async function storeKeys(storage: IStorage<string>, keys: Uint8Array) {
+  await storage.set(Uint8Array.from(keys).toString());
+}
+
+async function loadKeys(storage: IStorage<string>): Promise<Uint8Array | null> {
+  const val = await storage.get();
+  return val ? Uint8Array.from(val.split(',').map((c) => parseInt(c))) : null;
+}
+
+export type InitXmtpClientOptions = Partial<Omit<ClientOptions, 'env' | 'privateKeyOverride'>>;
```

Comment:
Does `Partial<...>` here risk to mislead consumer to provide configuration that is from from XMTP client perspective?

Reply:
this is the original type of xmtp options so I just omitted the two that we take care of

```ts
type InitClientArgs = {
    keys?: Uint8Array;
    options?: Partial<ClientOptions> & OnError;
    signer?: Signer | null;
};
```